front/lib/assistant/configuration_service.ts

```diff
@@ -0,0 +1,299 @@
+import type { Transaction } from "sequelize";
```

Comment:
_service is no a term we use.

Should live in lib/api/assistant/*

Reply:
Should actualy likely be in @app/lib/api/assistant/configuration next to createAgentActionConfiguration and createAgentConfiguration